src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -182,6 +182,19 @@
       </IncludedFileForRunnerScript>
     </ItemGroup>
     
+    <!-- Introduce workaround for Dependency list to prevent copying local assemblies now that test csproj references pkgproj
+      This was added to allow the version of buildtools the the buildtools repo depends on to build could be updated.
+      -->
+    <ItemGroup>
+      <IncludedFileForRunnerScriptRemove Include="@(IncludedFileForRunnerScript)"
+                                   Condition="$([System.String]::Copy('%(IncludedFileForRunnerScript.PackageRelativePath)').StartsWith('$(BinDir.Substring(0, $(BinDir.IndexOf('/'))))'))" >
```

Comment:
@weshaggard Included this line to prevent local dependencies from showing up in dependency list, which would break Helix runs. I started observing this after we started building the pkgproj reference in to the tests. This is a temporary workaround. Does this look ok at the moment?